docs/ubproject.toml

```diff
@@ -50,17 +50,17 @@ variant_options = ["author", "status"]
 
 # allowed values for need status core field
 # see https://sphinx-needs.readthedocs.io/en/latest/configuration.html#needs-statuses
-[[needs.statuses]]
-name = "open"
-description = "Open"
+# [[needs.statuses]]
```

Comment:
I have deactivated this, as it makes trouble and stops some needs from beeing created.